### DISCUSSION TITLE
variable = date d'aujourd'hui?

### DISCUSSION THREAD

**Discussion by florentarneodo** [MTESSDev/FRW#126] [Annonces]
Bonjour,

On aimerait afficher une date = aujourd'hui + 1 an dans un texte.

Comment est-ce possible?  On a l'habitude de travailler avec des dates que l'on demande, mais cette fois, on veut juste la date du jour automatiquement pour faire nos calculs.

Merci et bonne journée.

**Comment by j0ber** [comment]
Bonjour,
Voir exemple 4 ici : https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionStyle

Dans votre cas il suffit de mettre ce que vous voulez entre les accolades doubles. 
Ce n'est pas un code super beau mais si vous voulez un one liner en js pour votre besoin :  

```
- type: dynamic
  tag: div
  text:
    fr: |
      La date du jour est : {{new Date(new Date().setFullYear(new Date().getFullYear() + 1)).toISOString().slice(0, 10)}} 
```

Vous pourriez aussi créer une computed voir https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionInjecterJs qui retournerait la date souhaitée, et mettre cette computed dans les accolades, au lieu du code inline.

**Reply by florentarneodo** [reply]
merci!
